packages/scripts/src/admin-proposals/addContractCreator.js

```diff
@@ -158,25 +163,14 @@ async function run() {
     }
 
     // Send the proposal
-    console.group(`\n📨 Sending to governor @ ${mainnetContracts.governor.options.address}`);
+    console.group(`\n📨 Sending to proposer @ ${mainnetContracts.proposer.options.address}`);
     console.log(`- Admin proposal contains ${adminProposalTransactions.length} transactions`);
```

Comment:
Should you put both of these logs inside the proposeAdminTransactions method?

Reply:
done